source/documentation/io/streaming-io.md

```diff
@@ -87,7 +87,7 @@ an `StreamRDF` backed by a stream-based writer
 which can be done as:
 
     StreamRDF writer = StreamRDFWriter.getWriterStream(output, lang) ;
-    StreamOps.graphToStream(writer, model.getGraph()) ;
+    StreamRDFOps.graphToStream(writer, model.getGraph()) ;
```

Comment:
@afs this line is not in #22 , but the other line is in #22.

Reply:
`StreamRDFOps` is the right name so when this is merged, it'll be right.